drivers/arm/gic/v2/gicv2_main.c

```diff
@@ -593,3 +593,11 @@ unsigned int gicv2_set_pmr(unsigned int mask)
 
 	return old_mask;
 }
+
+/*******************************************************************************
+ * This functions configures the single interrupt trigger type
+ ******************************************************************************/
+void gicv2_interrupt_set_cfg(unsigned int id, unsigned int trigger_type)
```

Comment:
nit: would prefer cfg instead of trigger_type to match the API name. If you're reworking that, please also fix the comment.